admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.js

```diff
@@ -261,6 +261,8 @@ export default class OrderProductRenderer {
       $linkPagination.find('span').html(i);
       $linkPaginationTemplate.before($linkPagination.removeClass('d-none'));
     }
+
+    this.updatePaginationControls();
```

Comment:
ping @sowbiba can you validate the merge was done correctly ?

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/22655/commits/2a6b24c19f8e9b885e2cc48b25d319549afaf884